ui/tx/assetFlows/components/NovesTokenTransferSnippet.tsx

```diff
@@ -0,0 +1,46 @@
+import { Flex } from '@chakra-ui/react';
+import React from 'react';
+
+import NftEntity from 'ui/shared/entities/nft/NftEntity';
+import TokenEntity from 'ui/shared/entities/token/TokenEntity';
+
+interface Token {
```

Comment:
Shouldn't this type be derived from the `TokenInfo` then, if it is used as a prop for `TokenEntity` component?